<ui version="4.0" >
 <class>SymbolsUi</class>
 <widget class="QDialog" name="SymbolsUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>221</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPB" >
       <property name="toolTip" >
        <string>Insert the delimiters</string>
       </property>
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="categoryLA" >
       <property name="toolTip" >
        <string>Available categories depend on the document encoding. Select UTF-8 for the full range.</string>
       </property>
       <property name="text" >
        <string>Ca&amp;tegory:</string>
       </property>
       <property name="buddy" >
        <cstring>categoryCO</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="categoryCO" >
       <property name="maxVisibleItems" >
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>191</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="categoryFilterCB" >
       <property name="toolTip" >
        <string>Select this to display all available characters at once</string>
       </property>
       <property name="text" >
        <string>&amp;Display all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QListView" name="symbolsLW" >
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLineEdit" name="chosenLE" >
     <property name="readOnly" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="9" />
 <tabstops>
  <tabstop>applyPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_helpers.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
